common/workunit/workunit.cpp

```diff
@@ -7020,11 +7020,17 @@ bool extractFromWorkunitDAToken(const char * distributedAccessToken, StringBuffe
 //   1 : Signature does not verify (wuid/username don't match, or signature does not verify)
 //   2 : Workunit not active
 //   Throws if unable to open workunit
-wuTokenStates verifyWorkunitDAToken(const char * distributedAccessToken)
+wuTokenStates verifyWorkunitDAToken(const char * ctxUser, const char * daToken)
 {
+    if (isEmptyString(ctxUser) || isEmptyString(daToken))
```

Comment:
What happens if there is no security - ctxUser will be empty.

Reply:
This method is not called if there is no security manager